contrib/ansible/deploy-devel-playbook.yml

```diff
@@ -25,9 +25,17 @@
     machine_controller_image: "registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator:latest"
     machine_controller_image_pull_policy: "Always"
 
+    use_real_images: False
```

Comment:
What do you think about a rename for this var to something like "fake_deployment" and inverting the booleans? use_real_images could possibly slide under someone's visual radar, not realizing what that implies.